.github/workflows/run-linting.yml

```diff
@@ -0,0 +1,12 @@
+name: Run Linter
+
+on:
+  pull_request:
+    branches:
+      - main
```

Comment:
this repo has a `master` branch instead of `main`. also, can the branches be specified as an array instead (e.g. `branches: [master, ...]`?